etcd3/client.py

```diff
@@ -761,7 +887,7 @@ def create_alarm(self, member_id=0):
     def list_alarms(self, member_id=0, alarm_type='none'):
         """List the activated alarms.
 
-        :param: member_id:
+        :param member_id:
```

Comment:
Thanks for fixing this kind of typo, but it's even better if you do a PR just for that.